src/query/graphite/common/basic_functions.go

```diff
@@ -107,14 +108,17 @@ func Identity(ctx *Context, name string) (ts.SeriesList, error) {
 	}
 	newSeries := ts.NewSeries(ctx, name, ctx.StartTime, vals)
 	newSeries.Specification = fmt.Sprintf("identity(%q)", name)
-	return ts.SeriesList{Values: []*ts.Series{newSeries}}, nil
+	return ts.SeriesList{
+		Values:   []*ts.Series{newSeries},
+		Metadata: block.NewResultMetadata(),
+	}, nil
```

Comment:
nit: For this does it make sense to just create a special func like `ts.NewSeriesListWithSeries(...)` which auto creates the NewResultMetadata?